main.py

```diff
@@ -1,26 +1,28 @@
 ### Assignment: Regex and Files
-##Name:
-##Date: 
+##Name: Marcus Vardy
+##Date: 2/22/2021
 
 #don't forget to import regex
-
+import re
 
 ##(/5) Task 1: MODIFY the code below.
 #A# Change the regex from .* to what is required to capture an email address
 #B# Add a condition so that no blanks are printed.
-#C# Count the number of email addresses found and print a final output line.
+#C# Count the number of email addresses found and print a (final output line.)???
 
 def reEmail(fname):
-  
-  fhand = open(fname,'r')
-  for line in fhand:
-    extr = re.findall('.*',line)
-    if len(extr): 
-      print(extr)
+	count = 0
+	fhand = open(fname,'r')
+	for line in fhand:
+		extr = re.findall('.*@.*',line)
```

Comment:
This captures every character to the left and right of an @ sign, you want to stop when you reach a space.  ___    ajpauls@edu.pe.ca   ___

`extr = re.findall('\S+@\S+',line)`